CMakeLists.txt

```diff
@@ -192,7 +194,7 @@ if(ENABLE_ALL_WARNINGS)
   endif()
 endif()
 
-# Add our flags according to language
+# Add our extra flags according to language
 separate_arguments(NALU_CXX_FLAGS)
 separate_arguments(NALU_Fortran_FLAGS)
 target_compile_options(nalu PUBLIC $<$<COMPILE_LANGUAGE:CXX>:${NALU_CXX_FLAGS}>)
```

Comment:
We used to have the `NALU_CXX_FLAGS` printed out in the CMake output. It looks like that got lost during the CMake modernization refactor. Can we reinstate that?

Reply:
I actually removed that by choice. I found it really difficult to report the _final_ set of compiler flags during configure that CMake will actually use. Before, I was reporting the build type flags, Trilinos flags, and our own custom flags, but I would prefer to just report the final flags for C++ and Fortran. I could reinstate the old flag reporting, but I would like to find a better way. I found I could only rely on `CMAKE_EXPORT_COMPILE_COMMANDS` to tell me that.

Reply:
Since I call `target_compile_options` to add flags rather than build up a variable with all of them, which I could report, I was hoping there was some kind of reporting feature for it, but there doesn't seem to be. I'm still thinking about this, but I don't have a good solution. The old way of reporting started to become really incomplete after using `target_compile_options`.